### ISSUE TITLE
Website commanders don't use _adv basic weapon clones

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K#4795]
Website commanders are configured to use the basic module at level 3, so the UI drops this as invalid and leaves them with an unfilled slot. This is nominally an infra problem but perhaps it would be more mana-efficient to solve game-side.

**Comment by Licho1** [comment]
@sprunk  what exactly needs to be done?

**Comment by sprunk** [comment]
Commander config received from website has to have "_adv" appended to basic weapon names if they are in the level 4 slot, and they need to have a different cost there. This does not apply to basic weapons in level 2 slot, and also does not apply to non-basic weapons in the level 4 slot.

![image](https://user-images.githubusercontent.com/2573076/210258386-05fdb91f-1da1-448b-b5c3-95d62a2d80bf.png)

Also some modules have obsolete costs/descriptions but I don't have a list at hand, and it can be done via DB edits so I can bother semebody else later.

**Comment by Licho1** [comment]
Thanks, it's probably best to add in script generator or in game

**Comment by sprunk** [comment]
Yes, I think it should essentially be as simple as putting something like this in the website comm parser:
```lua
if level == 4 and isBasicWeapon(module) then
  module = module .. "_adv"
end
```
